tests/e2e/00-installation.spec.ts

```diff
@@ -20,6 +20,7 @@ const upMap: AppVersion[] = [
   { app: 'v1.8.0', controller: '2.0.0', crds: '1.4.2', defaults: '1.8.0' },
   { app: 'v1.9.0', controller: '2.0.5', crds: '1.4.4' }, // defaults: '1.9.2'
   { app: 'v1.10.0', controller: '2.0.8', crds: '1.4.5' }, // defaults: '1.9.3'
+  // { app: 'v1.11.0', controller: '2.0.9', crds: '1.4.5?' }, // defaults: '1.9.4'
```

Comment:
I imagine that you wanted to leave this commented out, correct?

Reply:
Yes, I will use it when we release 1.11.0.